README.md

```diff
@@ -105,6 +105,11 @@ Reference: [https://storychief.io/blog/chatgpt-prompts-seo](https://storychief.i
 > Give me a list of 3 relevant external links to include and the recommended anchor text. Make sure they’re not competing articles.<br/>
 > Split the outline into part 1 and part 2.
 
+## Act as an BlockChain Expert
```

Comment:
The Expected Output of the prompt, when entered systematically is:
![Capture](https://github.com/f/awesome-chatgpt-prompts/assets/133382057/f6d68f17-7bd0-4bef-be0c-87996dc38991)